components/micropython/vfs_sddf_fs_file.c

```diff
@@ -171,9 +171,9 @@ MP_DEFINE_CONST_OBJ_TYPE(
     locals_dict, &vfs_sddf_fs_rawfile_locals_dict
 );
 
-const mp_obj_vfs_sddf_fs_file_t mp_sys_stdin_obj = {{&mp_type_vfs_sddf_fs_textio}, STDIN_FILENO};
-const mp_obj_vfs_sddf_fs_file_t mp_sys_stdout_obj = {{&mp_type_vfs_sddf_fs_textio}, STDOUT_FILENO};
-const mp_obj_vfs_sddf_fs_file_t mp_sys_stderr_obj = {{&mp_type_vfs_sddf_fs_textio}, STDERR_FILENO};
+// const mp_obj_vfs_sddf_fs_file_t mp_sys_stdin_obj = {{&mp_type_vfs_sddf_fs_textio}, STDIN_FILENO};
+// const mp_obj_vfs_sddf_fs_file_t mp_sys_stdout_obj = {{&mp_type_vfs_sddf_fs_textio}, STDOUT_FILENO};
+// const mp_obj_vfs_sddf_fs_file_t mp_sys_stderr_obj = {{&mp_type_vfs_sddf_fs_textio}, STDERR_FILENO};
```

Comment:
?

Reply:
This was to change the STDIN, STDOUT and STDERR to be over serial, as defined in `sys_stdio_mphal.c`.

Reply:
Made this configurable by passing a flag in the makefile. Can choose between enabling vfs/serial stdio.